translations/de/hints/bryukh.html

```diff
@@ -1,10 +1,10 @@
 <body data-author="bryukh">
 <div class="question-answer">
     <div class="question">
-        I have no idea how to start solving this mission
+        Ich habe keine Ahnung wo ich mit der Lösung anfangen soll.
     </div>
     <div class="answer">
-        You can use a sequence of symbols or digraphs with values.
+        Du kannst eine Folge von Zeichen oder Digrafen mit Werten verwenden.
```

Comment:
It motivates at least to search for its meaning. And honestly, I get it now perfectly. 😁 Might be helpful, yes.